sdk/cmake/set_wine_module.cmake

```diff
@@ -0,0 +1,7 @@
+
+function(set_wine_module TARGET)
+    target_include_directories(${TARGET} SYSTEM BEFORE PRIVATE
+        sdk/include/reactos/wine
```

Comment:
Do we want (explicit) `${REACTOS_SOURCE_DIR}/` here?

Reply:
OK.